src/rulesets/CodeCop.ruleset.json

```diff
@@ -993,7 +993,8 @@
                   },
                   {
                       "id":  "AA0248",
-                      "action":  "Error"
+                      "action":  "None",
+                      "justification": "New rule for missing this. qualification on members. Turned off globally."
```

Comment:
perhaps "This rule contains too many false positives. It will soon be replaced with another rule."

Reply:
They will not be false positives and there will not be another rule, but you (app-team) must decide if you want to use this. qualification. We need a diagnostic to trigger the CodeFixer for it.

Reply:
Ok, then I misunderstood the justification for this. "New rule missing this" which new rule, missing what? "qualitifation on members", not sure what this means, so something about members doesn't work, but I don't understand what.

Reply:
Adding support a this keyword. You can optionally use it to qualify access to e.g. global vars/methods with this.MyGlobalVar := 42.